transmogrifier/import.rst

```diff
@@ -141,9 +141,13 @@ Here are some tips for debugging those errors.
    and is sometimes easier than working with the debugger to find the actual problem.
    Right before the traceback, you will see the output from the ``logger`` for the item being imported.
    You can grep through the entire export folder for the path output by the logger.
-   Tip: If you find a problem in the data, don't change the data in the export!
+
+.. tip::
+
+   If you find a problem in the data, don't change the data in the export!
    Unless you know for sure that you will not be exporting the data again,
    it's best to handle the issue in the import code.
```

Comment:
Above should be indented to match the indent of item 3 above, else it is outside the list and in a separate context.